File: app/src/main/java/com/qfq/tainzhi/videoplayer/mvp/presenter/impl/IIjkPlayerPresenter.java
package com.qfq.tainzhi.videoplayer.mvp.presenter.impl;

/**
 * Created by muqing on 2019/6/20.
 * Email: qfq61@qq.com
 */
public interface IIjkPlayerPresenter {
    void getSystemTime();
    
    void unRigister();
    
    void autoHidePanel();
    
    void getPlayerInfo(int roomId);
    
    void dismissVolAlpha();
}

File: app/src/main/java/com/qfq/tainzhi/videoplayer/mvp/presenter/impl/IIjkPlayerPresenter.kt
package com.qfq.tainzhi.videoplayer.mvp.presenter.impl

/**
 * Created by muqing on 2019/6/20.
 * Email: qfq61@qq.com
 */
open interface IIjkPlayerPresenter {
    open fun getSystemTime()
    open fun unRigister()
    open fun autoHidePanel()
    open fun getPlayerInfo(roomId: Int)
    open fun dismissVolumePanel()
}